air-script/tests/aux_trace/aux_trace.masm

```diff
@@ -1,3 +1,9 @@
+# Procedure to efficiently compute the required exponentiations of `z` and cache them for later use.
```

Comment:
nit: should we say something "... out-of-domain point `z` and ..."?

Reply:
changed